config.js

```diff
@@ -82,6 +82,7 @@ const config = {
         toBeReproduced: { name: 'TBR', automatic: true },
         toBeSpecified: { name: 'TBS', automatic: true },
```

Comment:
FYI these two labels don't exist anymore

Reply:
I update the code